content/developers/office_hours.md

```diff
@@ -0,0 +1,150 @@
+| [gae_datadog][38]                     | Fetch metrics from your GAE project and send them to Datadog                                                 | Outdated   |
+| [go-metro][39]                        | Passively calculate TCP RTT between hosts                                                                    | Active     |
+| [jmxfetch][43]                        | Export JMX metrics                                                                                           | Active     |
+| [omnibus-ruby][45]                    | Easily create full-stack installers for your project across a variety of platforms                           | Active     |
+| [universe][46]                        | The Mesosphere Universe package repository                                                                   | Active     |
+
+### Libraries 
+
+| Repository          | Description                          | Status   |
+| :--------           | :-------                             | :------- |
+| [dd-trace-py][9]    | Datadog Tracing Python Client        | Active   |
+| [dd-trace-go][10]   | A Go tracing package for Datadog APM | Active   |
+| [dd-trace-rb][11]   | Datadog Tracing Ruby Client          | Active   |
+| [dd-trace-java][12] | Datadog APM client for Java          | Active   |
+| [datadogpy][14]     | The Datadog Python library           | Active   |
+| [datadog-go][31]    | Go client library for datadog        | Active   |
+| [dogapi-rb][36]     | Ruby client for Datadog's API        | Active   |
+
+### DogStatsD
+
+| Repository                    | Description                                  | Status   |
+| :--------                     | :-------                                     | :------- |
+| [dogstatsd-csharp-client][13] | A DogStatsD client for C#/.NET               | Active   |
+| [php-datadogstatsd][20]       | A PHP client for DogStatsd                   | Active   |
+| [dogstatsd-ruby][37]          | A Ruby client for DogStatsd                  | Active   |
+| [java-dogstatsd-client][42]   | A java statsd client library                 | Active   |
+| [statsd-datadog-backend][46]  | A plugin to connect etsy's statsD to Datadog | Active   |
+
+
+### Miscellaneous
+
+| Repository                     | Description                                         | Status   |
+| :--------                      | :-------                                            | :------- |
+| [documentation][19]            | The source for Datadog's documentation site         | Active   |
+| [deb-s3][35]                   | Easily create and manage an APT repository on S3    | Outdated |
+| [gohai][40]                    | System information collector                        | Active   |
+| [heroku-buildpack-datadog][41] | Heroku Buildpack to run the Datadog Agent in a Dyno | Active   |
+| [node-connect-datadog][44]     | Datadog middleware for Connect JS / Express         | Active   |
+
+
```

Comment:
Let's remove some of the forks and outdated repos from this list.

Reply:
Should I remove all forked and outdated/deprecated repo?